event-input.c

```diff
@@ -2409,7 +2420,8 @@ evin_iomon_keypress_cb(mce_io_mon_t *iomon, gpointer data, gsize bytes_read)
     /* Power key press and release events count as actual non-synthetized
      *  user activity, but otherwise are handled in the powerkey module.
      */
-    if( ev->type == EV_KEY && ev->code == KEY_POWER ) {
+    if(( ev->type == EV_KEY && ev->code == KEY_POWER ) || 
+        (ev->type == EV_MSC && ev->code == MSC_GESTURE)) {
```

Comment:
Ditto, since type is not going to be changed to gesture -> this change becomes unnecessary -> undo